src/Nethermind/Nethermind.JsonRpc/JsonRpcService.cs

```diff
@@ -328,7 +328,11 @@ private void LogRequest(string methodName, string?[] providedParameters, Paramet
                     }
                     else
                     {
-                        executionParam = _serializer.Deserialize(new JsonTextReader(new StringReader($"\"{providedParameter}\"")), paramType);
+                        var stringReader = providedParameter.StartsWith('\"') && providedParameter.EndsWith('\"')
+                            ? new StringReader(providedParameter)
+                            : new StringReader($"\"{providedParameter}\"");
+                        var jsonTextReader = new JsonTextReader(stringReader);
+                        executionParam = _serializer.Deserialize(jsonTextReader, paramType);
```

Comment:
Why is that needed?

Reply:
for some reason it failed to deserialize already quoted strings